cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -88,6 +88,8 @@ public final class Constants {
    */
   public static final String APP_CDAP_VERSION = "app.cdap.version";
 
+  public static final String STREAMING_PLUGIN_TYPE = "streamingsource";
```

Comment:
Use io.cdap.cdap.etl.api.streaming.StreamingSource.PLUGIN_TYPE instead.

Reply:
To use this, I will have to add a dependency of cdap-etl-api-spark on cdap-app-fabric which is adding a circular dependency between modules cdap-app-fabric and cdap-etl-api.